### PR TITLE
[bitnami/minio] Allow Templates for Ingress Hostnames

### PR DESCRIPTION
**Pull request by FabianScheidt**
### Description of the change

I noticed that the ingress hostnames don't allow us to use template strings. I don't see a reason why this shouldn't be allowed, so this change will apply Helm's `tpl` function to both of them.

### Benefits

This will allow us to configure the following, which currently causes an error:

```yaml
    apiIngress:
        enabled: true
        hostname: '{{ .Release.Name }}-minio.example.com'
```

### Possible drawbacks

No known drawbacks.

### Applicable issues

Not reported yet.

### Additional information

None.

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/minio/Chart.yaml [hunk 1]
```diff
@@ -24,4 +24,4 @@ maintainers:
 name: minio
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/minio
-version: 12.6.4
+version: 12.6.5
```
Comment (andresbono) [inline bitnami/minio/Chart.yaml:27]: Could you please rebase your branch and bump the chart version again? Thanks!

**Review by andresbono — COMMENTED** [review]
Thank you for your contribution! Sorry for the late reply. Could you please check my comment?

File: bitnami/minio/Chart.yaml (hunk 1 above)
Comment (FabianScheidt) [reply bitnami/minio/Chart.yaml]: Sure! I have just updated the branch.

**Review by andresbono — APPROVED** [review]
LGTM. Thank you for your contribution!

**Pull request merged**